ethicalads-theme/templates/ea/landing-pages/data-science.html

```diff
@@ -0,0 +1,269 @@
+        </div>
+
+        <!-- Heading -->
+        <h3>Cookie free</h3>
+
+        <!-- Text -->
+        <p class="text-gray-800 mb-6 mb-md-0">We don't use cookies to follow users, nor any other trackers. Monetizing your traffic with us isn't selling out the privacy of your users. 
+
+      </div>
+      <div class="col-12 col-md-4 mb-5 mb-md-0 text-center" data-aos="fade-up" data-aos-delay="100">
+
+        <!-- Icon -->
+        <div class="icon text-primary mb-3">
+          <svg width="24" height="24" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="M0 0h24v24H0z"></path><path d="M16.5 4.5c-1.61 0-3.492 1.825-4.5 3-1.008-1.175-2.89-3-4.5-3C4.651 4.5 3 6.722 3 9.55c0 3.133 3 6.45 9 9.95 6-3.5 9-6.75 9-9.75 0-2.828-1.651-5.25-4.5-5.25z" fill="#335EEA" opacity=".3"></path><path d="M12 19.5C6 16 3 12.683 3 9.55 3 6.722 4.651 4.5 7.5 4.5c1.61 0 3.492 1.825 4.5 3v12z" fill="#335EEA"></path></g></svg>
+        </div>
+
+        <!-- Heading -->
+        <h3>Your data is yours</h3>
+
+        <!-- Text -->
+        <p class="text-gray-800 mb-0">Realtime data regarding your site's ads and revenue is available to browse and download at any time from your Publisher dashboard.</p>
+
+      </div>
+    </div> <!-- / .row -->
+  </div> <!-- / .container -->
+</section>
+
+
+<!-- Sample Ad -->
+<!-- ================================================== -->
+<section class="py-10">
+  <div class="container">
+    <div class="row justify-content-center">
+
+      <div class="col-md-6 mb-3">
+        <h2 class="font-weight-bold mb-8">Data driven and discreet ads</h2>
+        <p class="font-size-lg text-gray-800">We use the content of your pages to target advertising, so you'll only get ads that are relevent to your content. <strong>No ads that track your users around the internet.</strong> No ads that flash, move, or play music. Our ads complement your content rather than compete with it. You can even see every major advertiser that shows an ad on your site, right in your dashboard.</p>
+      </div>
+      <div class="col-md-6">
+        <!-- so the ad doesn't reflow the page -->
+        <div style="height: 280px;" class="text-center">
+          {% include 'ea/includes/ad-placement.html' %}
+        </div>
+        <!-- <p class="small text-muted text-center">(This is a real ad from our network)</p> -->
+      </div>
+
+    </div> <!-- / .row -->
+    {% with short=True %}
+    {% include 'ea/includes/advertiser-icons.html' %}
+    {% endwith %}
+  </div> <!-- / .container -->
+</section>
+
+<!-- Social proof -->
+<!-- ================================================== -->
+<section class="container py-10">
+  <div class="row">
+    <div class="col mb-3">
+      <h2 class="text-center font-weight-bold mb-8">Content with content-targeting</h2>
```

Comment:
This seems clever, but I don't think people will understand it. It took me a couple reads to understand it wasn't a typo. 

To keep a bit of alliteration, perhaps:

```suggestion
      <h2 class="text-center font-weight-bold mb-8">Proud of our privacy</h2>
```